src/Analyzers/MSTest.Analyzers/Resources.resx

```diff
@@ -509,4 +509,13 @@
   <data name="DynamicDataShouldBeValidMessageFormat_DataMemberSignature" xml:space="preserve">
     <value>'[DynamicData]' data member '{0}.{1}' signature is invalid</value>
   </data>
+  <data name="UseDeploymentItemWithTestMethodOrTestClassTitle" xml:space="preserve">
+    <value>'[DeploymentItem]' can be specified only on test class or test method</value>
+  </data>
+  <data name="UseDeploymentItemWithTestMethodOrTestClassDescription" xml:space="preserve">
```

Comment:
The description is supposed to be the place where you can be more exaustive. In that case, it adds nothing to the message format so I'd just skip the description.